datasets/_externals/brightquery.yml

```diff
@@ -0,0 +1,58 @@
+type: external
+title: BrightQuery Business Identity
+entry_point: zavod.runner.enrich:enrich
+disabled: false
+hidden: false
+coverage:
+  frequency: daily
+  start: 2025-11-07
+exports:
+  - statistics.json
+  - entities.ftm.json
+  - statements.csv
+  - targets.simple.csv
+# deploy:
+#   memory: "4000Mi"
+#   disk: "20Gi"
+http:
+  retry_methods:
+    - GET
+    # a match in BrightQuery is a POST request
+    - POST
+load_statements: true
+ci_test: false
+summary: >-
+  The BrightQuery Business Identity API provides firmographic and legal entity level
+  information for millions of US organizations, enabling enriched organization profiles.
+description: |
+  BrightQuery (BQ) provides company and legal entity data for approximately 74 million
+  U.S. organizations. It includes corporate family structures, legal status, firmographics,
+  compliance checks, headcount, revenue, and other key attributes. BQ aggregates and normalizes
+  data from multiple sources, delivering accurate and up-to-date information for identity
+  verification, risk assessment, market analysis, and business intelligence.
+tags:
+  - sector.financial
+url: https://permid.org/about
+publisher:
+  name: BrightQuery
+  acronym: BQ
+  description: |
+    BrightQuery is a data provider specializing in business identity and firmographic
+    information for U.S. organizations.
+  url: https://brightquery.com/about/
+
+inputs:
+  - us_dc_exclusions
+
+config:
+  type: nomenklatura.enrich.brightquery:BrightQueryEnricher
+  api_token: ${BQ_API_TOKEN}
```

Comment:
Pass in USER and PASS here.

Also, could we name them `BRIGHTQUERY_`? I think I'll forget about what BQ meant in two weeks.